### PR TITLE
Added CLI functions to create a new database including the System Administrator user

### PR DESCRIPTION
**Pull request by mikethetike**
This PR creates a way to create the System Admin user when creating a database for the first time. This allows a system admin user to be created without the need for backend access.

### CODE REVIEW

File: src/bin.rs [hunk 1]
```diff
@@ -25,23 +43,59 @@ pub fn main() {
...
     let parts: Vec<&str> = conn_string.split("/").collect();
     let db = parts.last().unwrap();
     let db = str::replace(db, "'", "''");
     let postgres_conn_string = str::replace(conn_string, &db, "postgres");
     let connection = PgConnection::establish(&postgres_conn_string).unwrap();
 
-    match connection.execute(&format!("CREATE DATABASE \"{}\"", db)) {
+    connection
+        .execute(&format!("CREATE DATABASE \"{}\"", db))
+        .map(|i| ())
```
Comment (CjS77) [inline src/bin.rs]: What does this do?

Comment (mikethetike) [reply src/bin.rs]: I wanted the `create_db()` to return "void" instead of the query results, which would not mean very much to the caller.

Comment (mikethetike) [reply src/bin.rs]: This basically swaps the result out for another result, or returns the error if there is one

**Comment by mikethetike** [comment]
@CjS77 The original intention was to not allow someone to add themselves to an existing db as an admin - only new installations. That said, if the user has a connection string with enough permissions, then they would be able to add themselves manually anyway...

I suggest leaving it as is and then split it if the need arises

**Pull request merged**